build-tools/xaprepare/xaprepare/Main.cs

```diff
@@ -308,5 +310,36 @@ static bool ParseBoolean (string value)
 
 			throw new InvalidOperationException ($"Unknown boolean value: {value}");
 		}
+
+		static RefreshableComponent ParseRefreshableComponents (string refreshList)
+		{
+			if (String.IsNullOrEmpty (refreshList))
+				return RefreshableComponent.None;
```

Comment:
This check could be split into two - check for `null` and if it's a match, return `None`; if the string is empty, however, return `All`

Reply:
Switched this over to use an optional argument as suggested. If no value is provided this function will never be hit, and the default value of `None` will be used. If `-refresh` or `-refresh=` is provided, `all` will be used.